Gopkg.lock

```diff
@@ -1,6 +1,26 @@
 # This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
 
 
+[[projects]]
+  digest = "1:dc62e76fd82a4bd369b92e8adc26d69d18c46d194f183179ac06c92fa4360472"
+  name = "github.com/Pallinder/go-randomdata"
```

Comment:
Do you know why this dependency is getting added?

It seems like it shouldn't be necessary for the tracer to work. It would be good to root it out so we don't add an unnecessary dependency to consumers of the package.

Reply:
Sorry, stray source file in the directory gets its dependencies picked up by dep.

Reply:
Fixed.